lib/sinon/util/fake_server.js

```diff
@@ -137,13 +142,21 @@ var fakeServer = {
         }
     },
 
+    logger: function () {
+        // no-op; override via configure()
```

Comment:
Ah, now I get it. I was confused by the `Set` construction mentioned further up. I glanced at what values were used there and noticed that it used booleans, and assumed they were not used for that purpose, but now that I know the values are not used things are suddenly looking different :-)